sympy/core/sympify.py

```diff
@@ -273,6 +298,40 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     >>> kernS(s)
     -2*(-(-x + 1/x)/(x*(x - 1/x)**2) - 1/(x*(x - 1/x))) - 1
 
+    Parameters
+    ==========
+
+    a :
+        - any object defined in SymPy
+        - standard numeric python types: int, long, float, Decimal
+        - strings (like "0.09", "2e-19" or 'sin(x)')
+        - booleans, including ``None`` (will leave ``None`` unchanged)
+        - dict, lists, sets or tuples containing any of the above
+
+    convert_xor : boolean, optional
+        If true, treats XOR as exponentiation.
+        If False, treats XOR as XOR itself.
+        Used only when input is a string.
+
+    locals : any object defined in SymPy, optional
```

Comment:
In order to have the sympy object be recognized it can be imported into a namespace dictionary and passed as locals. Like in the example of bitcount. I can provide an explanation here.

Reply:
I have changed the description now.